package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "get-npm-registry-package",
-  "version": "1.0.5",
+  "version": "1.0.6",
```

Comment:
don't update the version, I always do that using `npm version` so that changing the version get it's own commit

Reply:
going to fix this myself and release this